src/pyflex/window_selector.py

```diff
@@ -232,12 +247,12 @@ def _parse_event_and_station(self):
         # Last resort, if either is not set, and the observed or synthetics
         # are sac files, get the information from there.
         if not self.station or not self.event:
-            if hasattr(self.observed.stats, "sac"):
-                tr = self.observed
-                ftype = "observed"
-            elif hasattr(self.synthetic.stats, "sac"):
+            if hasattr(self.synthetic.stats, "sac"):
```

Comment:
Why did you invert that? I would tend to trust the observed data more than the synthetic sac headers.